File: Hospital.Api/Hospital.Api/Models/equipomedico.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Hospital.Api.Models
{
    public class equipomedico
    {
        [Key]
        public int idequipo { get; set; }
        public string nombre { get; set; }
        public string proveedor { get; set; }
        public int cantidad { get; set; }
    }
}

File: WEB/HOSPITEC/src/app/models/administracion/equipo-medico.ts
export class EquipoMedico {
    idequipo: string
    nombre: string;
    proveedor: string;
    cantidad: string;
  
    constructor(ideq, nom, prov, cant) {
      this.idequipo = ideq;
      this.nombre = nom;
      this.proveedor = prov;
      this.cantidad = cant;
    }
  
  }